frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/elements/IGraphElement.java

```diff
@@ -26,7 +26,12 @@
 
   GraphEdgeColor getGraphEdgeColor();
 
+  int getIndentLevel();
+
   boolean hasBulletPoint();
 
+  /** The subelement is an indented element (branch or commit). */
```

Comment:
Hmm what's the relation b/w the subelement and `downElementIndex`?

Reply:
There is no direct relation between them. subelement is a subbranch (or a first commit of subbranch) - sub may be understood as **indented**.  "Down element" is a graph element (commit/branch) that is below the given. If some subelements exist they "move" the `downElementIndex` down. `downElementIndex` refers to **unindented** elements (same indent level).

Reply:
Hmmm to keep the naming consistent-ish... what about `child`/`sibling`?

Reply:
done or at least read for **round two** of review :boxing_glove: